src/app/Processor/ProcessorDetail/ProcessorDetail.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { customRender, waitForI18n } from "@utils/testUtils";
+import ProcessorDetail from "./ProcessorDetail";
+import { SinkProcessor, SourceProcessor } from "../types";
+
+describe("ProcessorDetail component", () => {
+  it("should display sink processor information", async () => {
+    const comp = customRender(<ProcessorDetail processor={sinkProcessor} />);
+    await waitForI18n(comp);
+
+    expect(comp.getByText("Processor type")).toBeInTheDocument();
+
+    expect(comp.getByTestId("processor-type-label")).toHaveTextContent("Sink");
+
+    expect(comp.queryByText("Filters")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[0].key as string)
+    ).toBeInTheDocument();
+    expect(comp.queryByText("String contains")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[0].value as string)
+    ).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[1].key as string)
+    ).toBeInTheDocument();
+    expect(comp.queryByText("String equals")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[1].value as string)
+    ).toBeInTheDocument();
```

Comment:
in my opinion we should improve this tests. the `toBeInTheDocument` seems to me quite "easy to pass". 

With #16, we introduced locators for table, so now we should be able to check that something like

that component with `data-ouia-component-id='Filters'` contains `String equals` and `String contains`.

Reply:
Honestly I try to use `getByTestId` or `querySelector` as little as possible with RTL unit tests.

I agree with their principle of testing UI components as the user would use them, by looking at text or roles instead of ids or classes. They suggest to use `getByTestId` only when other types of queries are not possible.
In this case the `getByText` is sufficient to be sure the filters are rendered.

Do you suggest to rely to ids on a regular basis?

Reply:
@manstis _We agreed to write minimal tests_ yes, and I think my comment has no conflict with that. I am not asking to cover every single corner case. I am just asking to improve slightly the test, as the current `toBeInDocument()` assertion is quite easy to pass. 

@kelvah _Do you suggest to rely to ids on a regular basis?_ well, my point is not to stop searching by text, please lets have a look on these two examples:
- `get("data-id='filters-container'").contains("String equals")`
- `getDocument().contains("String equals")`

The first one is using some `data-id` attribute, but to me as reader of the tests gives much more information, what the "String equals" probably is.

From my point of view this is a way, how to improve test readability, for example for QE, who has no much opportunities to do actual development and we know the codebase less than DEV team.

What is your opinion please?

Reply:
As I said, it seems reasonable.

Reply:
I'm not convinced. But let's talk about this elsewhere, we are discussing testing guidelines / strategy and I think they are out of the scope of this PR.

I've updated the filters tests following your proposal. Let me know if you think they're OK now.